BridgeEmulator/web-ui/src/Room.js

```diff
@@ -81,31 +81,37 @@ const Room = ({
       </ListSubheader>
     }
   >
-    {room.lights.map(id => ({ ...lights[id], id })).map(light => (
-      <ListItem>
-        <ListItemIcon>
-          <IconButton
-            onClick={() => {
-              setDialogOpen(true);
-              setSelectedLight(light);
-            }}
-          >
-            {light.state.on ? (
-              <LightbulbOnIcon color="#FFF000" />
-            ) : (
-              <LightbulbOutlineIcon />
-            )}
-          </IconButton>
-        </ListItemIcon>
-        <ListItemText primary={light.name} />
-        <ListItemSecondaryAction>
-          <Switch
-            checked={light.state.on}
-            onChange={() => setState(light, !light.state.on)}
-          />
-        </ListItemSecondaryAction>
-      </ListItem>
-    ))}
+    {room.lights
+      .filter(id => Boolean(lights[id]))
+      .map(id => ({ ...lights[id], id }))
```

Comment:
We may get the rooms before we finished to fetch the lights. This would lead to an inconsistent state where we try to fetch informations about a specific light that is not yet available in the UI